File: Orientação a Objetos/TPC++/classes_cpp/funcionario.cpp
using namespace std;

#include "../headers/funcionario.hpp"


//Getters

int Funcionario::getId(){
    return this->id;
}

string Funcionario::getNome(){
    return this->nome;
}

string Funcionario::getSenha(){
    return this->senha;
}

string Funcionario::getUsername(){
    return this->username;
}

//Setters

void Funcionario::setId(int id){
    this->id = id;
}

void Funcionario::setNome(string nome){
    this->nome = nome;
}

void Funcionario::setSenha(string senha){
    this->senha = senha;
}

void Funcionario::setUsername(string username){
    this->username = username;
}

File: Orientação a Objetos/TPJAVA/TPJava/src/tpjava/Funcionario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tpjava;

/**
 *
 * @author nicolas
 */
public abstract class Funcionario {
    private int id;
    private String nome;
    private String username;
    private String senha;

    public Funcionario(int id, String nome, String username, String senha) {
        this.id = id;
        this.nome = nome;
        this.username = username;
        this.senha = senha;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }

    
    
    
    
}
